### PR TITLE
Added GitHub action to bump kubectl version

### PR DESCRIPTION
**Pull request by AverageMarcus**
Automate the bumping of this image to the latest stable release.

### CODE REVIEW

File: .github/workflows/update_version.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+on:
+  schedule:
+    - cron: "0 9 */7 * *"
+jobs:
+  debug_info:
+    name: Debug info
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Print github context JSON
```
Comment (whites11) [inline .github/workflows/update_version.yaml:9]: Why do we need this?

Comment (AverageMarcus) [reply .github/workflows/update_version.yaml:9]: I was following from the ones that had been auto-gen. Happy to remove if you think it's not needed.

Comment (whites11) [reply .github/workflows/update_version.yaml:9]: if we don't use it I'd remove it, but your call 100%

**Review by whites11 — APPROVED** [review]

File: .github/workflows/update_version.yaml (hunk 1 above)
Comment (AverageMarcus) [reply .github/workflows/update_version.yaml:9]: I'm gonna leave it for now while making sure the job actually works as expected. If all good then I'll remove :)

**Pull request merged**